docs/source/coding_guidelines.rst

```diff
@@ -0,0 +1,71 @@
+Coding Style Guidelines
+=======================
+
+DataTransferKit developers follow a set of style guidelines and use the clang
+format tool to create a consistent appearance to all source code committed to
```

Comment:
did the `format-cpp` rule exist?
* If not configure with `-D DataTransferKit_ENABLE_ClangFormat=ON`
* if so, it should have written "OK" to your stdout which means the code was fine in the first place